inventory/linux.cf

```diff
@@ -65,9 +65,19 @@ bundle common inventory_linux
       "has_proc_1_cmdline" expression => fileexists("/proc/1/cmdline"),
       comment => "Check if we can read /proc/1/cmdline";
 
+      "inventory_have_python_symlink" expression => fileexists("$(sys.bindir)/cfengine-selected-python");
+
     has_proc_1_cmdline::
       "systemd" expression => strcmp(lastnode($(proc_1_process), "/"), "systemd"),
       comment => "Check if (the link target of) /proc/1/cmdline is systemd";
+
+    inventory_have_python_symlink::
+      "cfe_python_for_package_modules_supported" -> { "CFE-2602", "CFE-3512", "ENT-10248" }
+        comment => "Here we see if the version of python found is acceptable for
+                    the package modules. We use this guard to prevent errors
+                    related to missing python modules.",
```

Comment:
```suggestion
        comment => concat( "Here we see if the version of python found is",
                           " acceptable ( version 2.4 or greater ) for package",
                           " modules. We use this guard to prevent errors",
                           " related to missing python modules."),
```